src/Cli/dotnet/commands/dotnet-tool/update/ToolUpdateLocalCommand.cs

```diff
@@ -77,71 +77,29 @@ public override int Execute()
             (FilePath? manifestFileOptional, string warningMessage) = 
```

Comment:
It looks like the only difference between this `Execute` method and the one for ToolInstallCommand is that this one may print a warning message.  Can we add that to the ToolInstallCommand and then remove the duplication by having this one call that `Execute` method?

Reply:
Resolved in #35088